tests/Unit/NumericalAlgorithms/RootFinding/Test_TOMS748.cpp

```diff
@@ -167,8 +178,12 @@ SPECTRE_TEST_CASE("Unit.Numerical.RootFinding.TOMS748",
         const DataVector lower{sqrt(2.0) - abs_tol, sqrt(2.0), -2.0, -3.0};
 
         const DataVector constant{2.0, 4.0, 2.0, 4.0};
-        const auto f_lambda = [&constant](const double x, const size_t i) {
-          return constant[i] - square(x);
+        const auto f_lambda = [&constant](const auto x, const size_t i) {
+          if constexpr (simd::is_batch<std::decay_t<decltype(x)>>::value) {
+            return simd::load_unaligned(&constant[i]) - square(x);
+          } else {
+            return constant[i] - square(x);
+          }
```

Comment:
[random line] maybe add a benchmark somewhere? You optimized this code, so it would be nice to be able to benchmark it. https://github.com/catchorg/Catch2/blob/devel/docs/benchmarks.md